src/cmd/netrcparser.cpp

```diff
@@ -97,14 +97,10 @@ bool NetrcParser::parse()
     }
 }
 
-NetrcParser::LoginPair NetrcParser::find(const QString &machine)
+NetrcParser::LoginPair NetrcParser::find(const QString &machine) const
 {
-    QHash<QString, LoginPair>::const_iterator it = _entries.find(machine);
-    if (it != _entries.end()) {
-        return *it;
-    } else {
-        return _default;
-    }
```

Comment:
How about just https://doc.qt.io/qt-5/qhash.html#value-1

Reply:
Right, I was concentrated on non-const find and non-const iterator. 
It's pretty the same underhood, but your variant is more readable.